wiki/Tournaments/UKCC/11/en.md

```diff
@@ -47,8 +47,73 @@ The UK Community Cup 11 is run by various community members.
+| #14 | **Mindmeister107** | ![][flag_HK] [mindmaster107](http://osu.ppy.sh/users/8819496), ![][flag_GG] [WoodMeister](http://osu.ppy.sh/users/10758866) |
+| #15 | **Deflated squidward 2** | ![][flag_GB] [-Deeline](http://osu.ppy.sh/users/10121815), ![][flag_GB] [[Nathan]](http://osu.ppy.sh/users/4741293) |
+| #16 | **stun seed** | ![][flag_GB] [Fwuffy](http://osu.ppy.sh/users/7374212), ![][flag_GB] [ZhiYo](http://osu.ppy.sh/users/9512841) |
+| #17 | **The** | ![][flag_GB] [lystia](http://osu.ppy.sh/users/11042418), ![][flag_GB] [Kingling](http://osu.ppy.sh/users/7010761) |
+| #18 | **the southern service to ore** | ![][flag_GB] [flubb](http://osu.ppy.sh/users/8211340), ![][flag_GB] [Xifan](http://osu.ppy.sh/users/9271492) |
+| #19 | **frogy** | ![][flag_GB] [p4905](http://osu.ppy.sh/users/11839754), ![][flag_GB] [Mahmood](http://osu.ppy.sh/users/7627844) |
+| #20 | **Meal Deal Men** | ![][flag_GB] [NeoPixel201](http://osu.ppy.sh/users/12139352), ![][flag_GB] [Tesco Meal Deal](http://osu.ppy.sh/users/6471972) |
+| #21 | **reshamen** | ![][flag_GB] [flag\_sc](http://osu.ppy.sh/users/17706086), ![][flag_GB] [Doughy](http://osu.ppy.sh/users/5275937) |
+| #22 | **language analysis spray** | ![][flag_GB] [uee](http://osu.ppy.sh/users/3793976), ![][flag_GB] [GongoTheGecko](http://osu.ppy.sh/users/9227012) |
+| #23 | **Mario Party Enjoyers (not Gaz)** | ![][flag_GB] [Gaz](http://osu.ppy.sh/users/3648459), ![][flag_GB] [Kathalshame](http://osu.ppy.sh/users/4572097) |
+| #24 | **osu!UK Lovers** | ![][flag_GB] [Teejay](http://osu.ppy.sh/users/9499753), ![][flag_GB] [Cephus](http://osu.ppy.sh/users/3373206) |
+| #25 | **bri ish** | ![][flag_GB] [Yozora](http://osu.ppy.sh/users/10085767), ![][flag_GB] [HonourBound](http://osu.ppy.sh/users/2945999) |
+| #26 | **Less-tah** | ![][flag_GB] [tekteks](http://osu.ppy.sh/users/11094254), ![][flag_GB] [Abielove](http://osu.ppy.sh/users/9346516) |
+| #27 | **Plympton** | ![][flag_GB] [Komm](http://osu.ppy.sh/users/7671790), ![][flag_GB] [Greninja](http://osu.ppy.sh/users/8579186) |
+| #28 | **sith is a slvt** | ![][flag_GB] [-jordan-](http://osu.ppy.sh/users/7288862), ![][flag_GB] [Sith](http://osu.ppy.sh/users/8112797) |
+| #29 | **Some Femboy Shit** | ![][flag_GB] [Kwichi-](http://osu.ppy.sh/users/16287466), ![][flag_GB] [Fluxey](http://osu.ppy.sh/users/9098192) |
+| #30 | **Platform Duty** | ![][flag_GB] [idealnettles](http://osu.ppy.sh/users/4010354), ![][flag_GB] [- Syndi -](http://osu.ppy.sh/users/11617315) |
+| #31 | **cheese gaming** | ![][flag_GB] [SinqHD](http://osu.ppy.sh/users/15887523), ![][flag_GB] [CrypticInsanity](http://osu.ppy.sh/users/11508900) |
+| #32 | **PopKO** | ![][flag_GB] [Claus](http://osu.ppy.sh/users/4966424), ![][flag_GB] [SpiralKO](http://osu.ppy.sh/users/5387803) |
+
 ## Mappools
 
+### Round of 32
+
+**[Download the mapack here (105 MB)](https://www.dropbox.com/s/yaoob4h4vmy8edy/UKCC%2011%20Round%20of%2032.zip?dl=0)**
```

Comment:
```suggestion
**[Download the mappack here (105 MB)](https://www.dropbox.com/s/yaoob4h4vmy8edy/UKCC%2011%20Round%20of%2032.zip?dl=0)**
```

Reply:
Of course the one gosh darn `Download the mappack here` line I copied had to be the only one with a mistake in it...

Reply:
where did you copy it from? O.o

Reply:
https://github.com/ppy/osu-wiki/blob/91765271b7f41e8b94aa595d48c89820430adaff/wiki/Tournaments/OWC/2021/en.md?plain=1#L248

Reply:
Deserves me right for not using the template...

Reply:
i guess it wasn't so bad, since you managed to find the *one* instance of this in the entire wiki